pkg/config/config.go

```diff
@@ -57,15 +58,17 @@ func (c Config) Blockchain() Blockchain {
 }
 
 // Load attempts to load the config from the given
-// path for the given netMode.
-func Load(path string, netMode netmode.Magic) (Config, error) {
+// path for the given netMode. If relativePath is not empty, relative paths in the
+// config will be updated based on the provided relative path.
+func Load(path string, netMode netmode.Magic, relativePath ...string) (Config, error) {
 	configPath := fmt.Sprintf("%s/protocol.%s.yml", path, netMode)
-	return LoadFile(configPath)
+	return LoadFile(configPath, relativePath[0])
```

Comment:
s/`relativePath[0]`/`relativePath...`

Consider the case when no `relativePath` is provided. Then panic occurs.